apps/examples/board_specific_demo/ameba_secure_storage_demo/secure_storage.c

```diff
@@ -51,6 +51,11 @@ static uint32_t SLOT1_START_ADDR = 0x1d000;
 static uint32_t SLOT0_START_ADDR = 0x24000;
 static uint32_t SLOT1_START_ADDR = 0x27000;
 #endif
+#ifdef CONFIG_AMEBASMART_TRUSTZONE
+/* ToDo: Temporary set need changes based flash addr */
+static uint32_t SLOT0_START_ADDR = 0x200000;
+static uint32_t SLOT1_START_ADDR = 0x203000;
```

Comment:
Could you let me know what they are?

Reply:
This is the temporary address for Secure Storage, so that the test case can be passed.